velox/common/memory/CMakeLists.txt

```diff
@@ -33,13 +33,13 @@ add_library(
 
 target_link_libraries(
   velox_memory
-  velox_flag_definitions
-  velox_common_base
-  velox_exception
-  velox_test_util
-  re2::re2
-  Folly::folly)
-
-if(NOT VELOX_DISABLE_GOOGLETEST)
```

Comment:
Theres was a conditional dependency on gtest - is that via velox_test_util now ?

Reply:
This was via ` GTestMacros.h` which was refactored to no longer include gtest so this can be removed (iiuc).